packages/grid_client/src/high_level/twinDeploymentHandler.ts

```diff
@@ -7,6 +7,7 @@ import {
   TimeoutError,
   ValidationError,
 } from "@threefold/types";
+import groupBy from "lodash/fp/groupBy.js";
```

Comment:
`lodash` is not in the client's dependencies and no need to add it just to group the deployments.

Reply:
I'm just using groupBy and imported only it's file might be ~ 1 KB at most
also lodash is installed somewhere and shared by lerna 
should we implement our own groupby and avoid using it here?

Reply:
yes, cause the client is published alone without the playground which has it.